src/webapp/components/upload/Completed.tsx

```diff
@@ -15,7 +17,7 @@ export const Completed: React.FC = () => {
             <p className="intro">{i18n.t("Thank! your data for now is uploaded in our system")}</p>
             <div className="ds-name">
                 <span>2020</span>
-                <span>{i18n.t("Spain")}</span>
+                <span>{i18n.t(currentOrgUnitAccess.orgUnitName)}</span>
```

Comment:
Not something to translate.

Reply:
Can a country name can be translated, for eg : in English "Spain" can be translated to "Espania" in Spanish? This is trying to solve that use case. Is it incorrect understanding?